pkg/defaults/defaults.go

```diff
@@ -41,6 +42,17 @@ func New() Defaults {
 	return &defaults{}
 }
 
+func (d *defaults) FilterDefaultOpSrc(in *v1.OperatorSource) {
```

Comment:
Missing doc. Moreover rather than filtering this seems to be `RestoreSpecIfDefault()`

Reply:
Good point, that's a better name than the one I thought of. Name updated and doc added in latest commit.